modules/apps/roll-dialog/cast-dialog.js

```diff
@@ -13,6 +13,13 @@ export default class CastDialog extends SkillDialog {
         return options;
     }
 
+    constructor(fields, data, resolve, options) {
+        super(fields, data, resolve, options);
+
+        this.initialFields.overchannelling = this.fields.overchannelling ?? 0;
+        this.fields.overchannelling = this.fields.overchannelling ?? 0;
+    }
```

Comment:
Is there a reason this isn't part of an overridden `_defaultFields` function? See https://github.com/moo-man/WFRP4e-FoundryVTT/blob/master/modules/apps/roll-dialog/roll-dialog.js#L184

Reply:
I will test it out

Reply:
resolved